### PR TITLE
Release resume

### PR DESCRIPTION
**Pull request by VolodymyrButs**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "semi": false,
+    "singleQuote": true,
+    "tabWidth": 4,
+    "trailingComma": "es5",
+    "ecmaVersion": 11,
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "tsx": true
+    },
+    "overrides": [
```
Comment (andriibuts) [inline .prettierrc]: Let's remove `overrides` from this project as we probably aren't going to use `yml`

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "endOfLine": "lf",
+    "semi": false,
+    "singleQuote": true,
+    "tabWidth": 4,
+    "trailingComma": "es5",
+    "ecmaVersion": 11,
+    "sourceType": "module",
+    "ecmaFeatures": {
+        "tsx": true
```
Comment (andriibuts) [inline .prettierrc]: No need for `tsx` here as well

**Review by andriibuts — CHANGES_REQUESTED** [review]
The main part looks good, let's remove unnecessary code from prettier config

File: .prettierrc (hunk 1 above)
Comment (VolodymyrButs) [reply .prettierrc]: delete unnecessary codes

File: .prettierrc (hunk 2 above)
Comment (VolodymyrButs) [reply .prettierrc]: delete unnecessary lines

**Review by andriibuts — APPROVED** [review]

**Pull request merged**